database/migration/src/main/resources/changesets/migration/workflow_store/replace_empty_custom_labels.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE WORKFLOW_STORE_ENTRY
+SET CUSTOM_LABELS = "{}"
+WHERE CUSTOM_LABELS = "";
```

Comment:
When I test the migration by excluding changeset id ="custom_labels_not_null" and only keeping changeset id="custom_labels_not_null", I get an error. 
WorkflowManagerActor Workflow b5140dfb-41db-4824-b6f6-c08af1e51384 failed (during MaterializingWorkflowDescriptorState): Workflow input processing failed:
Workflow contains invalid labels JSON: Unexpected end-of-input at input index 0 (line 1, position 1), expected JSON Value:

^